frontend-mobile/lib/view/coffee_call_screen.dart

```diff
@@ -18,6 +19,19 @@ class CoffeeCallScreen extends StatelessWidget {
     return Scaffold(
       appBar: AppBar(
         title: const Text('Coffee Caller'),
+        actions: [
+          TextButton.icon(
+            style: TextButton.styleFrom(foregroundColor: Colors.white),
+            onPressed: () {
+              Navigator.push(
+                context,
+                MaterialPageRoute(builder: (context) => const SettingsScreen()),
```

Comment:
@t-animal do you have any opinion on named routes?

Reply:
In the past I've used generated named routes, but that was before the introduction of the new declarative `Router` api. Generated routes and the router api would be overkill at this point, but we should keep in mind that we may have to switch to it when we introduce more pages.